src/rsz/test/buffer_varying_lengths.ok

```diff
@@ -2,14 +2,16 @@
 [INFO RSZ-0058] Using max wire length 827um.
 [INFO RSZ-0037] Found 118 long wires.
 [INFO RSZ-0038] Inserted 161 buffers in 118 nets.
-[INFO RSZ-0039] Resized 184 instances.
-worst slack -0.266
-tns -28.551
+[INFO RSZ-0039] Resized 186 instances.
+worst slack -0.269
+tns -28.504
 [INFO RSZ-0094] Found 200 endpoints with setup violations.
 [INFO RSZ-0099] Repairing 200 out of 200 (100.00%) violating endpoints...
-[INFO RSZ-0041] Resized 844 instances.
+[INFO RSZ-0041] Resized 889 instances.
 [WARNING RSZ-0062] Unable to repair all setup violations.
 [INFO RSZ-0033] No hold violations found.
-worst slack -0.266
-tns -27.434
-No differences found.
+worst slack -0.269
+tns -27.525
+Differences found at line 95.
+    - i1-179 BUF_X16 + PLACED ( 0 3580000 ) N ;
+    - i1-179 BUF_X8 + PLACED ( 0 3580000 ) N ;
```

Comment:
There should be no diffs in the ok file

Reply:
Ah, missed that, will fix here and in other places if there are any